src/main/java/org/tikv/common/DefaultHostMapping.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 PingCAP, Inc.
```

Comment:
TiKV Project Authors?

Reply:
Hm, looks like all files are using PingCAP, Inc.

Reply:
Maybe we can change licenses all at once with a separate PR.

Reply:
Yep we should investigate in another pass. I will create an issue.

Reply:
... created as https://github.com/tikv/client-java/issues/194.

I cannot resolve this comment, weird.